recipes/jwt-cpp/all/conanfile.py

```diff
@@ -18,15 +18,22 @@ def _source_subfolder(self):
 
     def requirements(self):
         self.requires("picojson/1.3.0")
-        self.requires("openssl/1.1.1d")
+        self.requires("openssl/1.1.1f")
 
     def source(self):
         tools.get(**self.conan_data["sources"][self.version])
         extracted_dir = self.name + "-" + self.version
         os.rename(extracted_dir, self._source_subfolder)
 
+    def build(self):
+        for patch in self.conan_data["patches"][self.version]:
+            tools.patch(**patch)
```

Comment:
@uilianries could you be so kind to let me know if this is correct for a header-only recipe?